parameters/uat.bicepparam

```diff
@@ -1,70 +1,74 @@
 using '../main.bicep'
 
-//1-Key Vault parameters
+// 1. Key Vault Parameters
 param keyVaultName = 'BestBank-KV-uat'
 param enableRbacAuthorization = true
 param enableVaultForDeployment = true
 param enableVaultForTemplateDeployment = true
 param enableSoftDelete = true
-param keyVaultRoleAssignments= [
+param keyVaultRoleAssignments = [
   {
-    principalId: '25d8d697-c4a2-479f-96e0-15593a830ae5' // BCSAI2024-DEVOPS-STUDENTS-A-SP
-    roleDefinitionIdOrName: 'Key Vault Secrets User' //so that the SP can access the secrets such as the swa token
+    principalId: '25d8d697-c4a2-479f-96e0-15593a830ae5' // Service Principal
+    roleDefinitionIdOrName: 'Key Vault Secrets User'
     principalType: 'ServicePrincipal'
-    }
+  }
+  {
+    principalId: 'a03130df-486f-46ea-9d5c-70522fe056de' // Group
+    roleDefinitionIdOrName: 'Key Vault Administrator'
+    principalType: 'Group'
+  }
 ]
 
-// //2 - azure container-registry
-param containerRegistryName = 'bestbankContRegistryUat'
+// 2. Azure Container Registry Parameters
+param containerRegistryName = 'bestbankContRegistryUAT'
```

Comment:
why did yo uchange the name of the resgistry? put it back to bestbankContRegistryUat